.travis.yml

```diff
@@ -1,15 +1,20 @@
 language: python
 sudo: false
-python:
-    - 2.6
-    - 2.7
-    - 3.3
-    - 3.4
-    - pypy
-    - pypy3
+
+env:
+  - TOXENV=py26
+  - TOXENV=py27
+  - TOXENV=py32
+  - TOXENV=py33
+  - TOXENV=py34
+  - TOXENV=pypy
+  - TOXENV=pypy3
+
 install:
-    - pip install .
+  - travis_retry pip install tox
+
 script:
-    - python setup.py test -q
+  - travis_retry tox
```

Comment:
I just looked:  `zope.i18n` is the only ZTK package which uses `travis_retry` for the `tox` command itself, rather than for the install phase (where it is a sensible workaround for network issues).  Let's back it out there and here.